docker-compose.build.yml

```diff
@@ -0,0 +1,33 @@
+# Used solely for docker-compose build
+version: '3'
+services:
+  postgis-base:
+    image: kartoza/postgis:base-${DISTRO}-${IMAGE_VERSION}-${IMAGE_VARIANT}
+    build:
+      context: .
+      # Use yaml anchor for reusability
+      args: &build-args-anchor
+        DISTRO: ${DISTRO}
+        IMAGE_VERSION: ${IMAGE_VERSION}
+        IMAGE_VARIANT: ${IMAGE_VARIANT}
+        GENERATE_ALL_LOCALE: ${GENERATE_ALL_LOCALE}
+        POSTGRES_MAJOR_VERSION: ${POSTGRES_MAJOR_VERSION}
+        POSTGIS_MAJOR_VERSION: ${POSTGIS_MAJOR_VERSION}
+        POSTGIS_MINOR_RELEASE: ${POSTGIS_MINOR_RELEASE}
```

Comment:
Using YAML anchor because all the build args used in all stages must be the same.